src/modules/Callout.js

```diff
@@ -0,0 +1,79 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+import Box from 'grommet/components/Box';
+import Heading from 'grommet/components/Heading';
+import Paragraph from 'grommet/components/Paragraph';
+import Anchor from 'grommet/components/Anchor';
+
+const CLASS_ROOT = 'callout';
+const PALM_BREAKPOINT = 720;
+
+export default class Callout extends Component {
+  constructor(props) {
+    super(props);
+    this._handleResize = this._handleResize.bind(this);
+    this.state = {
+      thumbnailWidth: 270,
```

Comment:
Actually probably best to handle this with css rather then javascript. I was trying not to add custom css but media queries sound more performant than adding javascript event listeners. What do you think @phuson?